.github/workflows/python-package-conda.yml

```
@@ -38,12 +38,16 @@ jobs:
         run: |
           pytest
       - name: Pytest Coverage
-        if: ${{ !github.event.pull_request.head.repo.fork }}
         run: |
-          pytest --cov=credit tests/ | tee pytest-coverage.txt
-      - name: Pytest coverage comment
-        uses: MishaKav/pytest-coverage-comment@1bdbba85fb74a2fdc1ec66383acec1091610f82b # v1.1.57
-        if: ${{ !github.event.pull_request.head.repo.fork }}
+           pytest --cov --cov-branch --cov-report=xml
```

```suggestion
           pytest --cov=credit --cov-branch --cov-report=xml
```

I think you still want to specify credit here otherwise you'll grab the tests directory as well.